### DISCUSSION TITLE
Does pgcenter suppor all statistic of postgresql14?

### DISCUSSION THREAD

**Discussion by wasiualhasib** [lesovsky/pgcenter#114] [General]
pgcenter really a very nice tools you made. Want all statistics information listed below. Also looking for documentation of this tools to learn it more details. Where I can get more details about it?

pg_stat_activity,pg_stat_replication,pg_stat_replication_slots,pg_stat_wal_receiver,pg_stat_subscription,pg_stat_archiver,
pg_stat_bgwriter,pg_stat_wal,pg_stat_database,pg_stat_database_conflicts,pg_stat_all_tables,pg_stat_all_indexes,pg_statio_all_tables
pg_statio_all_indexes, pg_statio_all_sequences.

Waiting for your reply.

**Comment by lesovsky** [comment]
Please read README carefully, there is a [list](https://github.com/lesovsky/pgcenter#postgresql-statistics) of supported statistics.

The following ones are **not** supported by pgcenter:
- pg_stat_replication_slots
- pg_stat_wal_receiver
- pg_stat_subscription
- pg_stat_archiver
- pg_stat_bgwriter
- pg_stat_database_conflicts

When using "pgcenter top" use hotkeys to switch between statistics. Press 'h' to show help screen with all possible hotkeys.

**Comment by wasiualhasib** [comment]
Oh sorry for that. I liked your suggestion. i love what you captured at
pgcenter.
Try to add more feature if possible.
Where i can know about your upcoming feature of those tools what you made?
Is there any personal facebook or other group?

**Comment by lesovsky** [comment]
Hmm, you can use "watch" button to get notifications about pgcenter or follow me on Github

**Comment by lesovsky** [comment]
Btw, adding such views listed above is not supposed because they are not widely used in practice. Maybe archiver and bgwriter will be added in future.

**Reply by wasiualhasib** [reply]
As a DBA I feel WAL archive , checkpoint and bgwriter is the core feature of PostgreSQL. If we can monitor and understand  those thing live and clearly then it could be better to understanding what is actually happening inside. I already included pgcenter in all postgresql node, it is working well. **I feel you can add something like checkpoint, bgwriter and archiver activity then it will help me and others also**.  I appreciate about, all your nice project .

**Reply by lesovsky** [reply]
I am a DBA too, and all my practice experience told me that checkpoint and bgwriter stats are almost useless. Read [this](https://www.postgresql.org/message-id/flat/CAAKRu_avP%2Bb%2BKEbCe1K0mdKhbJ5gVx8YzV37p%2BukCGkqFY1HJA%40mail.gmail.com#14ec5003d12db6bc50eeeb9957efc48e) for explanation. Archiver stats are also is rarely used.

Finally I have not see a strong reasons to add all exotic stats views. The wish of one person is not the reason.
